tests/formats/zeek/2.out.txt

```diff
@@ -0,0 +1 @@
+conn	2015-03-05T14:25:14.419939Z	CogZFI3py5JsFZGik	-	-	-	-
```

Comment:
If this in `formats/zeek` should this be in zeek format?

Reply:
Good point.  I'll move formats/zeek to flatten/zeek.  Text and table are zeek formats in disguise.